### DISCUSSION TITLE
Anyway we can Flatter (turn PDF) text and other content into images?

### DISCUSSION THREAD

**Discussion by hilmanski** [Hopding/pdf-lib#1337] [Q&A]
Hi,
PDF-lib has  a 'flatten' function to convert form fields.

I'd like to know is there any function to completely "flatten" the whole PDF
will be including text, etc..

So the result will be text won't be selectable anymore
as if they're image.
But still in PDF format

**Comment by mordechaim** [comment]
I'd love to know as well
